mapmssql2008.c

```diff
@@ -690,10 +690,10 @@ static char *strstrIgnoreCase(const char *haystack, const char *needle)
 {
   char    *hay_lower;
   char    *needle_lower;
-  int     len_hay,len_need;
+  size_t     len_hay,len_need;
   int     t;
   char    *loc;
-  int     match = -1;
+  size_t     match = -1;
```

Comment:
that one is problematic. size_t is a unsigned type. Setting a negative value to it will cause it to be interpreted as std::numeric_limits<size_t>::max() + 1 + value.